combine_FOGannotations.m

```diff
@@ -317,39 +515,120 @@ function combine_FOGannotations(filename_rater1, filename_rater2, filename_combi
 
 % load the big agreement table if present
 if exist(filename_agreement_table, 'file')
-  agreement_t=readtable(filename_agreement_table, 'FileType', 'text', 'ReadVariableNames', 1, 'HeaderLines', 0);
-  % check if this file is already part of the agreement table
-  if any(strcmp(agreement_t.filename, name)) % update the info in the table
-    n=find(strcmp(agreement_t.filename, name));
-    agreement_t(n,:)=agreement_info;
-  else % add an extra row to the table
-    agreement_t=[agreement_t; agreement_info];
-  end 
+    agreement_t=readtable(filename_agreement_table, 'FileType', 'text', 'ReadVariableNames', 1, 'HeaderLines', 0);
+    % check if this file is already part of the agreement table
+    if any(strcmp(agreement_t.filename, name)) % update the info in the table
+        n=find(strcmp(agreement_t.filename, name));
+        agreement_t(n,:)=agreement_info;
+    else % add an extra row to the table
+        agreement_t=[agreement_t; agreement_info];
+    end
 else % create a new table
-  agreement_t=agreement_info;
+    agreement_t=agreement_info;
 end
 
 % save the table
 writetable(agreement_t, filename_agreement_table,  'FileType', 'text', 'Delimiter', '\t');
 
 
 %% HELPER FUNCTIONS
+% CleanAnn
+function annotationclean = CleanAnn(annotation)
```

Comment:
do not correct for language differences (users should always use the English ELAN version), but correct all variable names to only include lower cases (adapt this in the rest of the script too!)